CHANGELOG.rst

```diff
@@ -1,6 +1,15 @@
 Changelog
 ---------
 
+2.20.0 (unreleased)
+*******************
+
+Deprecation/Removal:
+
+- Python 2.6 is no longer officially supported (:issue:`1274`).
+  NOTE: Dropping "official support" in this case means we will not test
+  against Python 2.6 in CI, but the code is still expected to work with Python 2.6.
```

Comment:
If it ain't tested, it is broken.

If we don't test anymore, we may introduce some PY2.7+ syntax without realizing it. I'd just remove this note.

But I don't mind.

Reply:
Fair enough. I added the note because this case is different from others in that we're not updating the codebase. But `!tested == broken` is truthy, as you said. I've removed the note.